.github/workflows/compose_test.yaml

```diff
@@ -54,13 +54,15 @@ jobs:
     strategy:
       matrix:
         BE_VERSION: [v3, v4]
-        FEATURE_ENABLED: ['', true]
+        JOBS_ENABLED: ['', true]
```

Comment:
JOBS and ELASTIC can stay as one, as separating them will create redundant combinations